orc8r/lib/go/definitions/env.go

```diff
@@ -24,3 +27,13 @@ func GetEnvWithDefault(variable string, defaultValue string) string {
 	}
 	return value
 }
+
+// MustGetEnv returns the string value of the environment variable,
+// returns error if it doesn't exist
+func MustGetEnv(variable string) (string, error) {
+	value := os.Getenv(variable)
+	if len(value) == 0 {
+		return "", fmt.Errorf("%s env not found", variable)
+	}
+	return value, nil
+}
```

Comment:
"Must" prefix indicating we want to panic if it fails. Will panic pretty immediately if the env var isn't set, in all services, so better to be noisy and panic than swallow the err